docs/contributors/releasing.md

```diff
@@ -49,7 +53,14 @@ The script will generate a new markdown file in `website/blog` filled with a
 basic release template.
 
 You can fill in the number of closed issues from the last milestone, though you
-will need to make sure everything is included there.
+will need to make sure everything is included there. In most cases you can just
+add all the closed issues since the last milestone. Please also fill in any
+missing details like the author or author image.
```

Comment:
```suggestion
add all the closed issues since the last milestone.

Please also fill in any missing details like the author or author image.
```
or put it after the last section, since it has nothing to do with milestones.